README.md

```diff
@@ -866,8 +866,8 @@ The simplest way is to pass datamodule attribute directly to model on initializa
 
 ```python
 # ./src/train.py
-datamodule = hydra.utils.instantiate(config.datamodule)
-model = hydra.utils.instantiate(config.model, some_param=datamodule.some_param)
+datamodule = hydra.utils.instantiate(cfg.datamodule)
+model = hydra.utils.instantiate(cfg.model, some_param=cfg.datamodule.some_param)
```

Comment:
```suggestion
model = hydra.utils.instantiate(cfg.model, some_param=datamodule.some_param)
```
Originally this was actually referring to datamodule object parameter, and not datamodule cfg parameter (that's why there's an initialization of datamodule object in the previous line).